frontend/src/routes/Governance/policies/policy-details/PolicyTemplateDetailHooks.tsx

```diff
@@ -133,11 +146,29 @@ export function useFetchKyvernoRelated() {
           .map((item: any) => {
             // Items are always clusterName + '/' + uid
             const uid = item._uid.split('/').slice(-1)[0]
-            return { ...item, policyReport: reportMap[uid] }
+            const policyReport = reportMap[uid]
+            let compliant = ''
+            const policyKey = namespace ? `${namespace}/${name}` : name
+
+            for (const violationMapValue of ((policyReport['policyViolationCounts'] as string) ?? '').split('; ')) {
+              if (!violationMapValue.startsWith(policyKey + '=')) {
+                continue
+              }
+
+              if (Number(violationMapValue.split('=', 2)[1]) > 0) {
```

Comment:
Is there any case that violation number more than 1

Reply:
if it is more than one,,. we consider Violation is just one noncompliant? I thought it was 2 violation

Reply:
@yiraeChristineKim if there is 1 or more violations on the object, then the object is considered noncompliant. There isn't a count in the violations column of the related objects table. This matches what is displayed by ConfigurationPolicy.

Reply:
okie I will add counting violations on my PR thanks!!

Reply:
@yiraeChristineKim good point!